packages/code-du-travail-frontend/pages/theme.js

```diff
@@ -8,7 +8,7 @@ import parents from "unist-util-parents";
 import { Link, Router } from "../routes";
 import Search, { SearchQuery } from "../src/search/Search";
 import Categories from "../src/Categories";
-import themes from "../src/data/themes.json";
+import themes from "../../code-du-travail-data/dataset/themes-front.json";
```

Comment:
:warning: This is kind of dangerous :warning: 
In a monorepo environment you should go through parent hierarchy *in the source code*!
The sources shouldn't be aware of the monorepo environment.
If you need to access to another package, use the created symlinks 

```diff
-import themes from "../../code-du-travail-data/dataset/themes-front.json";
+import themes from "code-du-travail-data/dataset/themes-front.json";
```